packages/extension/webpack.config.js

```diff
@@ -53,6 +53,7 @@ const baseConfig = {
     extensions: ['.svg', '.ts', '.tsx', '.js', '.json'],
     alias: {
       'react-onesignal': false,
+      'react/jsx-runtime': 'react/jsx-runtime.js',
```

Comment:
Why do you need the alias here?

Reply:
Since we are going to remove the dependency, I'm removing this change as well.